design-library/src/components/BccAccordion/BccAccordion.css

```diff
@@ -0,0 +1,105 @@
+@layer components {
+    .bcc-accordion {
+        @apply rounded-lg bg-secondary shadow-sm;
+    }
+
+    .bcc-accordion-header {
+        @apply flex w-full px-3 bg-white flex-row rounded-lg items-center justify-between cursor-pointer;
+    }
+
+    .bcc-accordion-title-wrap {
+        @apply flex flex-row items-center
+    }
+
+    .bcc-accordion:has(.bcc-accordion-interactive) {
+        @apply border border-on-primary bg-white;
+    }
+
+    .bcc-accordion:has(.bcc-accordion-desktop) {
+        @apply bg-white shadow-none;
+    }
+
+    .bcc-accordion:has(.bcc-accordion-lg) {
+        & .bcc-accordion-content {
+            @apply px-4;
+        }
+    }
+
+    .bcc-accordion:has(.bcc-accordion-readonly) {
+        & .bcc-accordion-content {
+            @apply pt-0 text-secondary text-body-sm;
+        }
+    }
```

Comment:
I tend to agree with your reasoning! But my opinion has changed slightly since I've been working on an app with Africa as the target audience - here we see a lot of people don't use modern phones and are often using older Chrome versions or even Opera Mini. So I'm thinking a bit of those people that will also use the Brunstad apps. But very likely this component will still be usable, just the UI will be a bit degraded, so I'll leave it up to you 😊